assets/js/langchoose.js

```diff
@@ -84,6 +84,45 @@ function selectLanguage(lang) {
     });
 }
 
+// copyNextCodeBlock copies the contents of the currently active code block following the element,
+// elem, which is the "copy" icon being clicked.
+function copyNextCodeBlock(elem) {
+    // Select the next visible code block.
+    var codes = $(elem).closest(".mdl-tabs").siblings(".highlight:visible");
+    if (codes.length) {
+        // Create an invisible text block containing the text from the code block, copy it to the
+        // clipboard, and then delete the invisible textarea.
```

Comment:
Why not use [`clipboard-polyfill`](https://www.npmjs.com/package/clipboard-polyfill) like we do in the service?

https://github.com/pulumi/pulumi-service/blob/22dc0c6409f7a5f244c3c8871eaccd217e6b5b93/cmd/console2/src/app/settings/tokens/token-settings.component.ts#L70

```typescript
(clipboard as any).writeText(value);
```

And let the polyfill handle older browsers?